### PR TITLE
show project current size and cpu if availlable in mongo

### PR DESCRIPTION
**Pull request by FanchTheSystem**
image

### CODE REVIEW

File: manager2/src/app/admin/projects/projects.component.ts [hunk 1]
```diff
@@ -170,40 +170,51 @@ export class ProjectsComponent implements OnInit {
...
-                if(resp.length == 0) {
+                if (resp.length == 0) {
                     return;
                 }
                 let data = resp;
-                for(var i=0;i<data.length;i++){
-                    data[i].expire = new Date(data[i].expire);
+                for (var i = 0; i < data.length; i++) {
+                    if (!data[i].expire) {
+                        data[i].expire = new Date(data[i].expire);
```
Comment (osallou) [inline manager2/src/app/admin/projects/projects.component.ts:208]: ? If not expire you set a new date with expire

File: manager2/src/app/project/project.component.ts [hunk 2]
```diff
@@ -95,6 +86,29 @@ export class ProjectComponent implements OnInit {
 
     }
 
+    project_list() {
+        this.projectsService.list(false).subscribe(
+            resp => {
+                for (var i = 0; i < resp.length; i++) {
+                    if (!resp[i].expire) {
+                        resp[i].expire = new Date(resp[i].expire);
```
Comment (osallou) [inline manager2/src/app/project/project.component.ts:94]: Same as above

File: manager2/src/app/admin/projects/projects.component.ts (hunk 1 above)
Comment (FanchTheSystem) [reply manager2/src/app/admin/projects/projects.component.ts:208]: Yes sorry I was in a meeting when a wrote this code ;) 
I will fix it

Comment (FanchTheSystem) [reply manager2/src/app/admin/projects/projects.component.ts:208]: It is a copy/past error, the original commit with this condition is here : https://github.com/genouest/genouestaccountmanager/commit/43e9f04f6d837f5d4a1a209c4e5b41816a9a1690

**Pull request merged**